kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -1,3 +1,5 @@
+import { browser, os, deviceWithTouch } from './browserInfo';
```

Comment:
Lets import `device` and `isTouchDevice` instead here. See https://github.com/learningequality/kolibri/pull/12382/files#r1675763461